<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExceptionDialog</class>
 <widget class="QDialog" name="ExceptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>174</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Exception</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
     <item>
      <widget class="QLabel" name="mpIcon">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>ICON</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mpExceptionInfo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>This is where the exception goes.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="mpDetails">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mpDetailsButton">
       <property name="text">
        <string>show details</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mpOkButton">
       <property name="text">
        <string>ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>mpDetailsButton</sender>
   <signal>clicked()</signal>
   <receiver>mpDetails</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpOkButton</sender>
   <signal>clicked()</signal>
   <receiver>ExceptionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>661</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
